### PR TITLE
pr for image and input components

### PR DESCRIPTION
**Pull request by suryxks**
Features added in this pr:

- Responsive image
- round image

- input with validations

### CODE REVIEW

**Review by someshtiwarii — APPROVED** [review]
good to go !

File: components/input/input.css [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    outline: none;
+}
+.warning-text{
+    color: var(--warning-text);
+    margin-top: 0px;
+}
+.danger>input{
+    border:2px solid var(--danger-text);
+    outline: none;
+}
```
Comment (BelideNikhil) [inline components/input/input.css]: do not apply styles directly on the element itself as it will effect globally, try giving these a class name

**Review by BelideNikhil — APPROVED** [review]
Good to go!

File: components/input/input.css [hunk 2]
```diff
@@ -0,0 +1,42 @@
+@import url('/components/css/common.css');
+form{
+    display: flex;
+    flex-direction: column;
+    padding:2rem;
```
Comment (manojasarsa) [inline components/input/input.css:5]: Spacing between property and value is not given. You can beautify file to make the code consistent.

**Review by manojasarsa — APPROVED** [review]
Looks good to merge!

File: components/input/input.css (hunk 1 above)
Comment (suryxks) [reply components/input/input.css]: thank you nikil have added class name and will merge it

**Pull request merged**